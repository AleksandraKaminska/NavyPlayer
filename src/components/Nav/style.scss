@media (min-width: 870px) {
    nav {
        text-transform: uppercase;
        position: absolute;
        top: 2.5vh;
        left: .5vw;
        z-index: 1;
        a {
            margin: .6vh 1.5vw;
            display: inline-block;
        }
    }
    .nav {
        display: block;
        img {
            width: 50px;
            height: auto;
            vertical-align: middle;
        }
        .active {
            font-weight: 500;
        }
    }
}
