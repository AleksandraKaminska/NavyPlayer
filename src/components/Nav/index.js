import React from 'react'
import { NavLink } from 'react-router-dom'
import './style.css'

const Nav = () => (
  <nav className='nav'>
    <NavLink to='/'>
      <img src='/logo/Logomark_white-resized.png' alt='' />
    </NavLink>
    <NavLink to='/top' activeClassName='active'>What's New</NavLink>
    <NavLink to='/artist' activeClassName='active'>Artist</NavLink>
    <NavLink to='/search'>Search</NavLink>
  </nav>
)

export default Nav
