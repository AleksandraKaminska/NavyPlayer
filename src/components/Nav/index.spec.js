import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import Nav from './index'
import renderer from 'react-test-renderer'

describe('Nav', () => {
  it('should render self', () => {
    const rendered = renderer.create(
      <Router>
        <Nav />
      </Router>
    )
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
