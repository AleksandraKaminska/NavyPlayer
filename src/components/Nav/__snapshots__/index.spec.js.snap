// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render self 1`] = `
<nav
  className="nav"
>
  <img
    alt=""
    src="/assets/logo/Logomark_white-resized.png"
  />
  <a
    aria-current="page"
    className="active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    Explore
  </a>
  <a
    aria-current={null}
    href="/artist"
    onClick={[Function]}
  >
    Artist
  </a>
  <a
    aria-current={null}
    href="/search"
    onClick={[Function]}
  >
    <div
      className="ðŸ“¦pst_relative ðŸ“¦dspl_inline-flex ðŸ“¦h_32px ðŸ“¦box-szg_border-box"
    >
      <div
        className="ðŸ“¦h_32px ðŸ“¦w_32px ðŸ“¦ptr-evts_none ðŸ“¦pst_absolute ðŸ“¦dspl_flex ðŸ“¦just-cnt_center ðŸ“¦algn-itms_center ðŸ“¦box-szg_border-box"
      >
        <svg
          className="ðŸ“¦z-idx_3 ðŸ“¦w_12px ðŸ“¦h_12px ðŸ“¦box-szg_border-box"
          data-icon="search"
          style={
            Object {
              "fill": "white",
            }
          }
          viewBox="0 0 16 16"
        >
          <title>
            search
          </title>
          <path
            d="M15.55 13.43l-2.67-2.68a6.94 6.94 0 0 0 1.11-3.76c0-3.87-3.13-7-7-7s-7 3.13-7 7 3.13 7 7 7c1.39 0 2.68-.42 3.76-1.11l2.68 2.67a1.498 1.498 0 1 0 2.12-2.12zm-8.56-1.44c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z"
            fillRule="evenodd"
          />
        </svg>
      </div>
      <input
        aria-invalid={false}
        className="css-5ljhhe ðŸ“¦color_rgb150-152-161 ðŸ“¦fnt-fam_b77syt ðŸ“¦fnt-sze_12px ðŸ“¦f-wght_400 ðŸ“¦ln-ht_16px ðŸ“¦ltr-spc_0 ðŸ“¦w_280px ðŸ“¦h_32px ðŸ“¦pl_32px ðŸ“¦pr_10px ðŸ“¦bblr_20px ðŸ“¦bbrr_20px ðŸ“¦btlr_20px ðŸ“¦btrr_20px ðŸ“¦bg_19g9jz0 ðŸ“¦b-btm_none ðŸ“¦b-lft_none ðŸ“¦b-rgt_none ðŸ“¦b-top_none ðŸ“¦box-szg_border-box"
        disabled={false}
        placeholder="Search"
        spellCheck={true}
        type="text"
      />
    </div>
  </a>
</nav>
`;
